src/select/select.stories.ts

```diff
@@ -25,4 +25,20 @@ storiesOf("Select", module).addDecorator(
           </optgroup>
 		</ibm-select>
 	`
+	}))
+	.add("Inline", () => ({
+		template: `
+		<ibm-select inline="true">
```

Comment:
if you use `inline` here, the HTML Dom element for the Select component itself will have an extra attribute `inline` there. If you use `[inline]` instead, then the style will be applied at the right place without having the additional attribute added to the Dom note.